UITableView_pageSetting/ViewController.swift

```diff
@@ -8,12 +8,188 @@
+                        Cell(title: "Не беспокоить"),
+                        Cell(title: "Экранное время"),
+                       
+                        Cell(title: "Основное"),
+                        Cell(title: "Пункт управления"),
+                        Cell(title: "Экран и яркость"),
+                        Cell(title: "Экран домой")]
+        return Network
+    }
+}
+
+
+struct CellApiNotifications{
+    static func getDataNotifications() -> [Cell] {
+    let notifications = [Cell(title: "Уведомления"),
+                        Cell(title: "Звуки, тактильные сигналы"),
+                        Cell(title: "Не беспокоить"),
+                        Cell(title: "Экранное время")]
+    return notifications
+    }
+}
+
+struct CellApiGeneral{
+    static func getDataGeneral() -> [Cell] {
+    let general = [Cell(title: "Основное"),
+                   Cell(title: "Пункт управления"),
+                   Cell(title: "Экран и яркость"),
+                   Cell(title: "Экран домой")]
+    return general
     }
+}
 
+var allCellDataNetwork = CellApiNetwork.getNetworkTitle()
+var allCelldataNotifications = CellApiNotifications.getDataNotifications()
+var allCellDataGeneral = CellApiGeneral.getDataGeneral()
+let allCell = [allCellDataNetwork, allCelldataNotifications, allCellDataGeneral]
 
+extension ViewController: UITableViewDelegate {
+    
+    // MARK: клик на ячейку
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+        let title = allCell[indexPath.section][indexPath.row].title
+        print("Выбрана ячейка: \(title ??  "")")
+    }
+    
+    // MARK: Количество секций
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 3
+    }
+    
+    // MARK: Header and Footer sections
+    public func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        let view = UIView()
+        view.backgroundColor = UIColor.lightGray
+        return view
+    }
+    
+    public func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return 5
+    }
+    
+    public func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let view = UIView()
+        view.backgroundColor = UIColor.lightGray
+        return view
+    }
+    
+    public func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 5
+    }
+    
+}
+
+extension ViewController: UITableViewDataSource{
+    
+    // MARK: Количество строк в секции
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return 6
+        } else if section == 1 {
+            return 4
+        } else if section == 2 {
+            return 4
+        } else {
+            return 0
+        }
```

Comment:
не забывай про оператор switch - согласись лаконичнее?
```suggestion
        switch section {
        case 0:
            return 6
        case 1:
            return 4
        case 2:
            return 4
        default:
            return 0
        }
```